### DISCUSSION TITLE
Contact Form on Jekyll ?

### DISCUSSION THREAD

**Discussion by salimbenfarhat** [community/community#45803] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, it is possible to integrate gem(plugin) on "jekyll/github pages" to create customized contact form on ".md" or ".html" file, without passing by external services like formspree.io or getform. Thanks you.

**Comment by jge162** [comment]
Check out this [Repository](https://github.com/anchetaWern/jekyll-contact)

**Reply by salimbenfarhat** [reply]
it's works with formkeep or pooleapp services

**Comment by salimbenfarhat** [comment]
Finally I've found solution by integrating JS code to send results form on gsheet row and after automatically its transfered on my email with Google Apps Script.

**Reply by djdanielsson** [reply]
an example would be awesome to help others looking for a similar solution

**Reply by salimbenfarhat** [reply]
@djdanielsson, my solution also works with Google Sheets and Apps Script to send emails. I'll post an example here as soon as I can.
